### ISSUE TITLE
There are spaces around operators most of the time, but not here

### ISSUE THREAD

**Issue by fuelen** [rrrene/credo#703]
### Environment

* Credo version (`mix credo -v`): 1.1.5
* Erlang/Elixir version (`elixir -v`): 1.8.1
* Operating system: Linux

Credo reports
```
There are spaces around operators most of the time, but not here
```
for the following line:
```
build_timeframe.(-30, 10),
```

**Comment by TheFirstAvenger** [comment]
@fuelen I was unable to reproduce this locally, is this still happening? If so, can you provide a larger code sample or a public repo with the minimal code that reproduces it?

**Comment by fuelen** [comment]
@TheFirstAvenger 
file `test.exs`
```elixir
defmodule TestTest do
  def test do
    a = fn b, c -> b + c end

    a.(-30, 10)
  end
end
```
```
$ mix credo test.exs
Checking 1 source file ...
  Consistency                                                                   
┃ 
┃ [C] ↗ There are spaces around operators most of the time, but not here.
┃       test.exs:5:10 #(TestTest.test)

Please report incorrect results: https://github.com/rrrene/credo/issues

Analysis took 0.07 seconds (0.01s to load, 0.06s running checks)
2 mods/funs, found 1 consistency issue.
```

**Comment by TheFirstAvenger** [comment]
@fuelen I was able to reproduce the issue on 1.1.5 and it went away with the latest master. Can you confirm the issue goes away for you if you upgrade to master?

**Comment by fuelen** [comment]
yes, thanks a lot!
